src/NUnitFramework/framework/Constraints/Operators/ExactCountOperator.cs

```diff
@@ -39,17 +39,24 @@ public class ExactCountOperator : CollectionOperator
         /// <param name="expectedCount">The expected count</param>
         public ExactCountOperator(int expectedCount)
         {
+            // Collection Operators stack on everything
+            // and allow all other ops to stack on them
+            this.left_precedence = 1;
+            this.right_precedence = 10;
+
```

Comment:
I'm uncertain... so maybe I want a test. :-)

ExactCountOperator was IIRC the last CollectionOperator added... long after the others. So it probably didn't get exhaustive syntactic tests. We should see what happens when it's followed by things like `And` and `Or`. That will give us a better idea of the priority to use. Take a look at the tests we have (I hope they are better) for some of the original CollectionOperators, like Some, No and All.